pandas/tests/groupby/test_any_all.py

```diff
@@ -178,3 +178,14 @@ def test_object_NA_raises_with_skipna_false(bool_agg_func):
     ser = Series([pd.NA], dtype=object)
     with pytest.raises(TypeError, match="boolean value of NA is ambiguous"):
         ser.groupby([1]).agg(bool_agg_func, skipna=False)
+
+
+@pytest.mark.parametrize("box", [DataFrame, Series])
```

Comment:
You can use the `frame_or_series` fixture

Reply:
Thanks!